package/gluon-check-connection/luasrc/usr/bin/gluon-check-connection

```diff
@@ -4,17 +4,11 @@ local unistd = require 'posix.unistd'
 local util = require 'gluon.util'
 local site = require 'gluon.site'
 
-local offline_flag_file = "/tmp/gluon_offline"
+local offline_flag_file = "/var/gluon/check-connection/offline"
 local is_offline = true
 
--- Check if domain-switch is scheduled
-if site.domain_switch() == nil then
-	-- Switch not applicable for current domain
-	os.exit(0)
-end
-
 -- Check reachability of pre-defined targets
-for _, ip in ipairs(site.domain_switch.connection_check_targets()) do
+for _, ip in ipairs(site.check_connection.targets()) do
 	local exit_code = os.execute("ping -c 1 -w 10 " .. ip)
```

Comment:
please add to the documentation that a node is considered online if any of the specified ip addresses can be reached via ping.

Reply:
done